src/style.css

```diff
@@ -1 +1,92 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+:root {
+  --text-color: #3d4e61;
+  --input-box-shadow: 0 1px 8px 0 #3d4e611a;
+  --box-shadow: 0 3px 8px 0 #3d4e6133;
+  --linear-gradient: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+body {
+  /* the big search bar must have top indent 20px */
+  margin-top: 20px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Setting `margin-top` here might lead to inconsistent spacing in the layout. Consider using only `margin-bottom` on elements.